sql/core/src/main/scala/org/apache/spark/sql/SparkSessionExtensions.scala

```diff
@@ -61,6 +61,26 @@ import org.apache.spark.sql.catalyst.rules.Rule
  *     .getOrCreate()
  * }}}
  *
+ * The extensions can also be used by setting the Spark SQL configuration property
+ * spark.sql.extensions, for example:
```

Comment:
nit: spark.sql.extensions -> `'spark.sql.extensions'`

Shall we also mention multiple rule can be set via comma separated?

Reply:
I added the recommended documentation updates and also added code marks around `withExtensions` and fixed the typo there. I didn't think it was necessary to provide the example of using the comma-separated string but I did note it in the documentation.